46.md

```diff
@@ -45,15 +45,25 @@ _remote-signer_ provides connection token in the form:
 bunker://<remote-signer-pubkey>?relay=<wss://relay-to-connect-on>&relay=<wss://another-relay-to-connect-on>&secret=<optional-secret-value>
 ```
 
-_user_ passes this token to _client_, which then sends `connect` request to _remote-signer_ via the specified relays. Optional secret can be used for single successfully established connection only, _remote-signer_ SHOULD ignore new attempts to establish connection with old secret. 
+_user_ passes this token to _client_, which then sends `connect` request to _remote-signer_ via the specified relays. Optional secret can be used for single successfully established connection only, _remote-signer_ SHOULD ignore new attempts to establish connection with old secret.
 
 ### Direct connection initiated by the _client_
 
-_client_ provides a connection token in the form: 
+_client_ provides a connection token using `nostrconnect://` as the protocol, and `client-pubkey` as the origin. Additional information should be passed as query parameters:
+
+- `relay` (required) - one or more relay urls on which the _client_ is listening for responses from the _remote-signer_.
+- `secret` (required) - a short random string that the _remote-signer_ should return as the `result` field of its response.
+- `perms` (optional) - a comma-separated list of permissions the _client_ is requesting be approved by the _remote-signer_
+- `name` (optional) - the name of the _client_ application
+- `url` (optional) - the canonical url of the _client_ application
+- `icon` (optional) - a small image representing the _client_ application
```

Comment:
Make it "image"? Nostrudel and Coracle use "image" as query-string param, nsec.app understands that and I assume Amber too. I only used "icon" in json metadata in nostr-login.